ws-nextjs-app/pages/[service]/send/[id]/FormField/InvalidMessageBox.tsx

```diff
@@ -43,33 +43,39 @@ const ErrorSymbol = () => (
   </svg>
 );
 
-export default ({
-  id,
-  messageCode,
-  hasArrowStyle = true,
-  suffix,
-}: InvalidMessageBoxProps) => {
-  const {
-    translations: { ugc = defaultUGC },
-  } = useContext(ServiceContext);
+const ErrorMessageBox = forwardRef(
+  (
+    { id, messageCode, hasArrowStyle = true, suffix }: InvalidMessageBoxProps,
+    ref: ForwardedRef<HTMLElement>,
+  ) => {
+    const {
+      translations: { ugc = defaultUGC },
+    } = useContext(ServiceContext);
 
-  const message = ugc[messageCode ?? InvalidMessageCodes.FieldRequired];
+    const message = ugc[messageCode ?? InvalidMessageCodes.FieldRequired];
 
-  return (
-    <>
-      {hasArrowStyle && <div css={styles.errorArrow} />}
-      <div css={styles.errorMessageBox(hasArrowStyle)}>
-        <ErrorSymbol />
-        <Paragraph
-          id={id}
-          css={styles.errorText}
-          fontVariant="sansBold"
-          size="minion"
+    return (
+      <>
+        {hasArrowStyle && <div css={styles.errorArrow} />}
+        <div
+          css={styles.errorMessageBox(hasArrowStyle)}
+          tabIndex={-1}
```

Comment:
This will need to be on an element that isn't a wrapper for AT to be able to focus on it, if we aren't adding the strong element as per the screen reader UX then it should be on the paragraph element